dbschema/budget.esdl

```diff
@@ -0,0 +1,19 @@
+module default {
+  type Budget extending Project::Child, Project::ContextAware {
+    required status: Budget::status {
+      default := Budget::status.Pending;
+    };
+
+    records: BudgetRecord;
```

Comment:
Ok so with this and above, there are two points of truth for the relationship.
We want this to be either
```
multi records: BudgetRecord;
```
or
```
records := .<budget[is BudgetRecord]
```
(same holds true for `BudgetRecord.budget` as well)

I don't know off the top of my head which is better, it's obv use case dependent.

Reply:
Let me put this out there as a proposed solution:

```
type Budget extending Project::Child {
    records := .<budget[is Budget::Record];
}

type Record extending Project::Child {
    required budget: Budget {
      on target delete delete source;
    };
}
```

Thoughts?

Reply:
Yeah makes sense to me! 

I'm trying to find the use case for the other way around. Where the multi link is the stored truth instead of a computed back link.
I guess `User.pins` is a good example

Reply:
In that situation is there even a backlink at all?